stumpy/Mmotifs.py

```diff
@@ -0,0 +1,297 @@
+import numpy as np
+import logging
+import stumpy
+import matplotlib.pyplot as plt

```

Comment:
We should remove this as this would add an unnecessary new user dependency to STUMPY. Currently, STUMPY only requires 3 core dependencies (i.e., `numpy`, `scipy`, and `numba`). We should not require that users install additional software in order for it to run.